content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -678,14 +678,15 @@ plugins:
 #### Configuration Annotation Format
 
 `PodNodeSelector` uses the annotation key `scheduler.alpha.kubernetes.io/node-selector` to assign
-node selectors to namespaces.
+node selectors to namespaces. This namespace configuration file describes a namespace that forwards a node selector to its pods, disktype: ssd. 
+This means that all pods of namespace namespace3 will get scheduled on a node that has a disktype=ssd label.
```

Comment:
Thanks for opening this PR, @ebuildy. But, I don't think this improvement is necessary because this implementation was clearly described [here](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#internal-behavior):
> If the `Namespace` has an annotation with a key `scheduler.alpha.kubernetes.io/node-selector`, use its value as the node selector.

Also, you are talking about a **label**, but the doc is talking about an **annotation**.

Reply:
yes you are right.